### PR TITLE
Feature/spa-experience

### PR DESCRIPTION
**Pull request by IgboPharaoh**
This introduces the wrapper for the single-page app experience for MVP.

[Link to design wrapper](https://www.figma.com/file/ioeAVbWBc3ow2RaZETatM7/Fedimint?node-id=5520%3A78245&mode=dev)

Pictorial representation

Screenshot 2023-07-21 at 21 20 35

### CODE REVIEW

**Comment by okjodom** [comment]
Some review notes:
- All strings in the footer component need to be sourced from the translation framework
- [follow-up] We should look to share this wrapper component and relevant shared elements with GatewayUI.

**Comment by IgboPharaoh** [comment]
> Some review notes:
> 
> * All strings in the footer component need to be sourced from the translation framework
> * [follow-up] We should look to share this wrapper component and relevant shared elements with GatewayUI.

- Corrections have been made for the strings using the translation framework

File: apps/guardian-ui/src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+          >
+            <Stack spacing='12px'>
+              <Text
+                fontSize={{ base: 'sm', sm: 'md', md: 'md', lg: 'md' }}
+                color={theme.colors.blue[50]}
+                fontWeight='600'
+              >
+                {t('footer.docs')}
+              </Text>
+              <CustomLink title='Getting Started' href='fedimint.org' />
```
Comment (okjodom) [inline apps/guardian-ui/src/components/Footer.tsx:62]: needs translation

Comment (IgboPharaoh) [reply apps/guardian-ui/src/components/Footer.tsx]: Please review changes have been made

File: apps/guardian-ui/src/components/Wrapper.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import React, { memo } from 'react';
-import { Box, Center } from '@chakra-ui/react';
+import { Box, Flex } from '@chakra-ui/react';
+import { Header } from './Header';
+import Footer from './Footer';
```
Comment (okjodom) [inline apps/guardian-ui/src/components/Wrapper.tsx:4]: nit: we should probably standardize our component exports. Non -

**Review by okjodom — APPROVED** [review]
LGTM

File: apps/guardian-ui/src/components/Wrapper.tsx (hunk 2 above)
Comment (IgboPharaoh) [reply apps/guardian-ui/src/components/Wrapper.tsx:4]: Alright, will make necessary corrections on the next issue

File: apps/guardian-ui/src/languages/en.json [hunk 3]
```diff
@@ -119,5 +119,22 @@
     "table_column_hash_input": "Paste verification code",
     "wait_all_guardians_verification": "Waiting for all Guardians to verify their codes",
     "all_guardians_verified": "All Guardians have verified their codes"
+  },
+  "footer": {
+    "docs": "Docs",
+    "community": "Community",
+    "contribute": "Contribute",
+    "getting_started": "Getting Started",
+    "fedimint_org_link": "https://fedimint.org",
```
Comment (okjodom) [inline apps/guardian-ui/src/languages/en.json:128]: I don't think we mean to translate these types of links in the UI

Comment (okjodom) [reply apps/guardian-ui/src/languages/en.json:128]: https://github.com/fedimint/ui/pull/112#issuecomment-1646688992

File: apps/guardian-ui/src/languages/en.json [hunk 4]
```diff
@@ -119,5 +119,22 @@
     "table_column_hash_input": "Paste verification code",
     "wait_all_guardians_verification": "Waiting for all Guardians to verify their codes",
     "all_guardians_verified": "All Guardians have verified their codes"
+  },
+  "footer": {
+    "docs": "Docs",
```
Comment (okjodom) [inline apps/guardian-ui/src/languages/en.json:124]: nit: we should make string labels more descriptive, to help future translators

**Pull request merged**